include/lmms_math.h

```diff
@@ -360,6 +360,21 @@ static inline int numDigitsAsInt(float f)
 	return digits;
 }
 
+static inline float minEpsilon()
+{
+	return 1.0e-10f;
```

Comment:
```suggestion
	return 1.0e - 10f;
```

Reply:
What broke?

Reply:
It is an exponential operation, not subtraction.

Reply:
Btw, would it make sense to use this value as a constant rather than a function?

Reply:
I converted this to a constant.

Reply:
I don't know what I am thinking sometimes...sorry

Reply:
> It is an exponential operation, not subtraction.

A constant here is fine too.